docs/application/web/guides/TOC.md

```diff
@@ -1,21 +1,25 @@
 # Application Management
 ## Applications
+### [Overview](app-management/applications.md)
 ### [Service Application](app-management/service-app.md)
 ### [Watch Application](app-management/watch-app.md)
 ### [Widget Application](app-management/web-widget.md)
-## [Application Groups](app-management/app-group.md)
-## [Application Preferences](app-management/preference.md)
-## [Application Icons](app-management/app-icons.md)
 ## Application Information and Controls
-### [Application Controls](app-management/app-controls.md)
-### [Common Application Controls](app-management/common-appcontrol.md)
-### [Widget Information](app-management/widget-app.md)
+### [Overview](app-management/app-info-control.md)
+### [Application Groups](app-management/app-group.md)
+### [Application Preferences](app-management/preference.md)
+### [Application Icons](app-management/app-icons.md)
+### [Application Controls](app-management/appcontrols.md)
+### [Common Application Controls](app-management/common-appcontrols.md)
+### [Widget Information](app-management/widget-info.md)
+### [Package Information](app-management/packages.md)
 ## Application Data Exchange
+### [Overview](app-management/app-communications.md)
```

Comment:
Current file name for app-communications.md is app-communication.md. s is missing.

Reply:
Done.